### PR TITLE
adding api trasient button

### LINKED ISSUE #128: Add download transient data button to the transient results pages
# **🚀 Feature Request**

## **Is your feature request related to a problem? Please describe.**

* We should have a way to download all the data from the API via a button on each transient results page

---

## **Describe the solution you'd like**

* Add a button to the transient results pages, which points to the API end point for that transient. 

This would mean adding code here after this line:

https://github.com/astrophpeter/blast/blob/8443d38382d18a51a7d6fe0a0d50c25e921f8d27/app/host/templates/host/results.html#L11

Hints:

* Consider a transient with name 2022test. The html link that the new button would have to point to would have to be
/api/transients/get/2022test

* The way to get a transient's name in the template is by using {{transient.name}}

### PR DESCRIPTION
**Pull request by Lila-braff**
Fixes #128   .

## Checklist

Please check all that apply to your proposed changes

- [x] HTML code change
- [ ] Added package dependency
- [ ] Added data
- [ ] Changed django model
- [ ] Documentation change
- [ ] Added or changed TaskRunner

### CODE REVIEW

File: app/host/templates/host/results.html [hunk 1]
```diff
@@ -8,9 +8,9 @@
 {% block body %}
 
 <div class="p-5 text-left bg-light">
-  <h1 class="mb-3">{{transient.name}}</h1>
+  <h1 class="mb-3">{{transient.name}} test </h1>
```
Comment (astrophpeter) [inline app/host/templates/host/results.html:11]: Nice work, I think you need to remove the word test from this line though  :)

Comment (Lila-braff) [reply app/host/templates/host/results.html:11]: I believe I fixed both issues and merged the changes successfully! let me know if you see it on your end.

File: app/host/templates/host/results.html [hunk 2]
```diff
@@ -8,9 +8,9 @@
 {% block body %}
 
 <div class="p-5 text-left bg-light">
-  <h1 class="mb-3">{{transient.name}}</h1>
```
Comment (Lila-braff) [inline app/host/templates/host/results.html:11]: got it

**Review by astrophpeter — APPROVED** [review]
Really awesome work, well done. Will merger shortly.

**Pull request merged**